app/views/data_files/edit.html.erb

```diff
@@ -29,7 +32,7 @@
 	:templates => authorised_assets(FileTemplate, nil, 'edit') } %>
       <% end %>
       
-      <%= render partial: 'assets/controlled_vocab_annotations_form_properties', :locals => { :resource => (@data_file ? @data_file : DataFile.new) } -%>
+      <%= render :partial => 'assets/controlled_vocab_annotations_form_properties', :locals => { :resource => (@data_file ? @data_file : DataFile.new) } -%>
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys